### PR TITLE
Get by name replaced by get by id

### PR DESCRIPTION
**Pull request by DrPandemic**

### CODE REVIEW

**Comment by bluespart** [comment]
I'm bluespart and I approve of this PR!

**Comment by MicLamb** [comment]
Things are progressing pretty quickly.

**Comment by Baltazar510** [comment]
Not to much modification, only some correction.
That good

File: backend/server.js [hunk 1]
```diff
@@ -26,83 +26,80 @@ var port = 8080;        // set our port
...
+      res.send(err);
+    else if(!hello) {
+      res.status(404);
+      res.json({message: "Hello not found"});
+    } else
+      res.json(hello);
+  });
+});
+
+router.route('/character')
```
Comment (JesseEmond) [inline backend/server.js]: unrelated: shouldn't this be `POST /characters`?
[this link](http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api) apears to agree

What do you think?

**Comment by Erikad94** [comment]
:+1:

File: backend/server.js (hunk 1 above)
Comment (JesseEmond) [inline backend/server.js]: Or should we use plural everywhere?
https://stackoverflow.com/questions/6845772/rest-uri-convention-singular-or-plural-name-of-resource-while-creating-it

**Pull request merged**